tsconfig.json

```diff
@@ -69,20 +69,14 @@
         // The source files of everything listed under `exports` in our package.json
         // (i.e. public API's that should be documented) should be listed here:
         "src/index.ts",
-        "src/index.ts",
+        "src/error.ts",
```

Comment:
```suggestion
```
This doesn't need to/shouldn't be here.

Reply:
Without this here, the docs don't generate on that file.

Reply:
That's because `src/index.ts` is excluded on line 83/79

Reply:
hahaha, this config is a mess ^^,

Reply:
Copied from an old version of the template. 🤷‍♂️